e2e/testinfra/terraform/prow/networks.tf

```diff
@@ -27,3 +27,93 @@ resource "google_compute_subnetwork" "e2e-subnetwork-1" {
   description = "Subnetwork for use in e2e test clusters"
   private_ip_google_access = false
 }
+
+resource "google_compute_subnetwork" "e2e-subnetwork-2" {
```

Comment:
How does the quota look like, do we have a limitation on the subnetworks

Reply:
All of these subnetworks fit within the quota, I've already ran this to set up the new test environment. I committed a bit of a faux pas by running this before getting it code reviewed, but I didn't want to have conflicting terraform definitions submitted on `main`.

Reply:
Got it. So in an ideal world the re-provision won't happen in a while? And the tf config aligns with the previous deployed setup?

Reply:
I didn't want to update these terraform configs until the migration was complete. The current configuration aligns with the legacy setup that is now being removed - my new changes reflect the new setup. I moved to a new project with the migration, so in the interim the previous project was still running with the legacy setup.

Reply:
Understood, thanks for the clarification.